dev/tools/localization/bin/gen_localizations.dart

```diff
@@ -65,8 +66,10 @@ String generateArbBasedLocalizationSubclasses({
   required String baseClass,
   required HeaderGenerator generateHeader,
   required ConstructorGenerator generateConstructor,
+  ConstructorGenerator? generateConstructorForCountrySubClass,
   required String factoryName,
   required String factoryDeclaration,
+  required bool callsFactoryWithConst,
```

Comment:
How does this affect the generated code?

Reply:
The const keyword in https://github.com/flutter/flutter/blob/3173f275015cfd2d35790f1c97de3a32235f130f/packages/flutter_localizations/lib/src/l10n/generated_widgets_localizations.dart#L2693

The generated material localization can't use const here because they pass in parameter here